.travis.yml

```diff
@@ -17,5 +17,8 @@ before_install:
     - GO111MODULE=off go get github.com/opencontainers/umoci/cmd/umoci
     - sudo cp $GOPATH/bin/umoci /usr/bin
     - curl -sSfL https://raw.githubusercontent.com/golangci/golangci-lint/master/install.sh | sh -s -- -b $(go env GOPATH)/bin v1.24.0
+    - sudo apt-get install -yy autoconf automake make autogen autoconf libtool binutils git squashfs-tools
+    - (cd /tmp && git clone https://github.com/AgentD/squashfs-tools-ng && cd squashfs-tools-ng && ./autogen.sh && ./configure --prefix=/usr && make -j2 && sudo make -j2 install && sudo ldconfig -v)
```

Comment:
squashfs has code to do this (to get squashfs-tools-ng and build it).

As root, you should be able to:

    git clone https://github.com/anuvu/squashfs && cd squashfs && 
        ./setup --install-packages /usr/local &&
        make && cp squashtool/squashtool /usr/local/

squashfs obviously has the need to maintain that path as working.  So that is one less thing for you to have to worry about.

Reply:
bah... to run the squashtool/squashtool you also need '/sbin/ldconfig'.
For your purposes, maybe 

    make static && cp squashtool/squashtool.static /usr/bin/squashtool

Reply:
thats fine... and fwiw, it wasn't clear above, but `--install-packages` would handle your 'apt-get' line too.